api/src/test/resources/OML_O21.hl7

```diff
@@ -1,4 +1,4 @@
 MSH|^!\&|iSantePlus||OpenELIS||20180330150703.345+{time_offset}00||OML^O21^OML_O21|2018033015070300000|D|2.5.1
-PID||1004X^^^^^2|||Doe^Joe^Terry|Janine||M|||Trou coucou^1ère L'Arbre^Anse Rouge^Artibonite^^HTI^^Anba Morne Porte|||||^Separe|^Vaudou||||||Birth City
+PID||1004X^^^^^2|||Doe^Joe^Terry|Janine|19870211101500+{time_offset}00|M|||Trou coucou^1ère L'Arbre^Anse Rouge^Artibonite^^HTI^^Anba Morne Porte|||||^Separe|^Vaudou||||||Birth City
```

Comment:
Lack of 13th section 'Phone Number - Home' (marked as green in Inbound (HL7 2.5.1 OML^O21)).